src/main/scala/li/cil/oc/integration/appeng/DriverBlockInterface.scala

```diff
@@ -15,13 +15,14 @@ import li.cil.oc.util.ExtendedArguments._
 import li.cil.oc.util.ResultWrapper._
 import net.minecraft.item.ItemStack
 import net.minecraft.util.EnumFacing
+import net.minecraft.util.math.BlockPos
 import net.minecraft.world.World
 
 object DriverBlockInterface extends DriverSidedTileEntity {
```

Comment:
`getInventoryByName`? That's part of `appeng.api.implementations.tiles.ISegmentedInventory`